tests/model/schema.js

```diff
@@ -1047,6 +1047,26 @@ describe( 'Schema', () => {
 					return true;
 				}
 			} );
+
+			schema.addAttributeCheck( ( ctx, attributeName ) => {
+				// Disallow 'italic' on $text that has attribute 'bold'.
+				if ( inTextWithBold( ctx ) && attributeName == 'italic' ) {
+					return false;
+				}
```

Comment:
Missing blank line.

Reply:
Right above there's another addAttributeCheck() call. Why haven't you extended it?